### ISSUE TITLE
6.1.0 generates bigger files

### ISSUE THREAD

**Issue by XhmikosR** [imagemin/imagemin#310]
The cause must be https://github.com/imagemin/imagemin/pull/285

/CC @sindresorhus

**Comment by XhmikosR** [comment]
I suggest that we revert the patch and release a new patch version ASAP since this is a serious regression.

**Comment by sindresorhus** [comment]
It was an intentional change. And IMHO a good one. What if the user has set to strip metadata and that's important for their use-case, but the resulting image is larger, so we use the original instead. That would could lead to privacy issues and it's surprising behavior.

**Comment by XhmikosR** [comment]
That's one case you describe. For most cases, this is inconsistent behavior. Running `jpegoptim` alone for example doesn't produce a bigger image.

**Comment by XhmikosR** [comment]
We need this fixed. How about an option? Right now we generate bigger files in the majority of cases, especially for jpeg files.

/CC @malash

**Comment by sindresorhus** [comment]
Sure. But the docs for it should clearly explain the downsides of using it.

**Comment by XhmikosR** [comment]
That's easy to do. I insist about this because it's a breaking change. Not sure when I'll be able to take a look so any help is welcome.

**Comment by XhmikosR** [comment]
I'm still hitting this issue for binary image formats, and I'm not sure how to proceed. I know for a fact after the aforementioned change, all users will get bigger files not only for SVGs, which the patch was supposed to fix, but even for JPEG, PNG etc.

Going into each imagemin plugin to do the same is one way, but it seems easier to just handle this here for SVGs only.

**Comment by malash** [comment]
Maybe the `--force` option is a good solution if user intents to use the bigger output ?

**Comment by XhmikosR** [comment]
Yeah, that would work, @malash. Note that for binary formats, I doubt anyone would want bigger files.

**Comment by mahish** [comment]
Im literally getting all of my images bigger than before in all projects. Simple JPG "optimalization". So I reverted back to old version. 

It is out of my capabilities to help you fix this but Im sure this is not the way it should work. Specially, when `jpegoptim` alone doesn't have this issue as mentioned @XhmikosR

**Comment by XhmikosR** [comment]
If anyone makes a patch, please CC me. I'd really like this fixed and the old behavior restored.

**Comment by TashaBasalt** [comment]
Was there ever a compromise for this? Or some instruction somewhere on which plugins won't make my files bigger? There isn't much in the docs, and as of now I can't use imagemin if it isn't going to actually give me smaller files.

**Comment by XhmikosR** [comment]
No fix was made AFAICT. I'm unsure how to solve this apart from reverting the patch...
